fr/tutorials-and-examples/blog-auth-example/auth.rst

```diff
@@ -63,7 +63,7 @@ responsabilité de trouver, sauvegarder et valider toute donnée d'utilisateur::
                 ->notEmpty('role', 'Un rôle est nécessaire')
                 ->add('role', 'inList', [
                     'rule' => ['inList', ['admin', 'author']],
-                    'message' => 'Merci d'entrer un rôle valide'
+                    'message' => 'Merci d\'entrer un rôle valide'
```

Comment:
```suggestion
                    'message' => "Merci d'entrer un rôle valide"
```

For better readability, avoid escaping by using single-quote in double-quoted string.